Assets/MRTK/Tests/PlayModeTests/SpatialObserverTests.cs

```diff
@@ -23,7 +23,7 @@ namespace Microsoft.MixedReality.Toolkit.Tests
     /// <summary>
     /// Test class that validates observers start and stop based on right configuration in editor. 
     /// </summary>
-    public class SpatialObserverTests : BasePlayModeTests
+    public class SpatialObserverTests
```

Comment:
Why the move away from the base class here?

Reply:
The base playmode test does an additional initialization of the MRTK scene, which then produces conflicts because the SpatialObserver tests are also making profile adjustments and trying to reinit the scene at the same time.

Our other ProfileTests class which does a similar thing does not derive from BasePlayModeTests, so I think it makes more sense to move away from the base class.